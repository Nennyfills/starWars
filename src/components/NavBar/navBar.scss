.nav {
  background-color: #834b73;
  padding: 0.1rem;
  color: #ffffff;
  box-shadow: 0 0 3px;
  .nav_fonts {
    margin: 1rem;
  }
  .nav-logo{
    font-size: 2rem;
  }
}
