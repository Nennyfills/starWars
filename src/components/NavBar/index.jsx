import React from 'react';

import './navBar.scss';

/**
 * NavBar Component
 *  @returns navBar element
 */
const NavBar = () => (
  <div className="nav">
    <h4 className="nav_fonts">
      <span className="nav-logo">S</span>
      tarWars
    </h4>
  </div>
);

export default NavBar;
