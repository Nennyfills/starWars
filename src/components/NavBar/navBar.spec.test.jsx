import React from 'react';

import NavBar from './index';

describe('NavBar Component', () => {
  it('should mount the NavBar wrapper', () => {
    const wrapper = mount(<NavBar />);

    expect(wrapper.find('.nav')).toBeDefined();
    expect(wrapper).toMatchSnapshot();
  });
});
