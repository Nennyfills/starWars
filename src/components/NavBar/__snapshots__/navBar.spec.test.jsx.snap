// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar Component should mount the NavBar wrapper 1`] = `
<NavBar>
  <div
    className="nav"
  >
    <h4
      className="nav_fonts"
    >
      <span
        className="nav-logo"
      >
        S
      </span>
      tarWars
    </h4>
  </div>
</NavBar>
`;
